buffer/src/test/java/io/netty/buffer/AdaptiveByteBufAllocatorTest.java

```diff
@@ -113,4 +117,38 @@ void sliceOrDuplicateUnwrapLetNotEscapeRootParent(boolean slice) {
 
         assertTrue(buffer.release());
     }
+
+    // See https://github.com/netty/netty/issues/15028
```

Comment:
No need to ref the issue here, got blame will tell us

Reply:
Done it.